backend/controllers/applications.controller.js

```
@@ -1,7 +1,11 @@
 import mongoose from 'mongoose';
 import jwt from 'jsonwebtoken';
 import Application from '../models/applications.js'; // Import Mongoose model
+import path from 'path';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)